test/cases/040_packages/030_logwrite/test.yml

```diff
@@ -0,0 +1,34 @@
+kernel:
+  image: linuxkit/kernel:4.14.53
+  cmdline: "console=ttyS0 console=ttyAMA0"
+init:
+  - linuxkit/init:6cc1442112980c889230b6449df09d5b48de6854
+  - linuxkit/runc:v0.4
+  - linuxkit/containerd:f2bc1bda1ab18146967fa1a149800aaf14bee81b
+  - linuxkit/ca-certificates:v0.4
+  - linuxkit/memlogd:883f0d46e7d3ae2d787e8acb496da115a4707cbc
+services:
+# A service which generates logs of log messages
+  - name: fill-the-logs
+    image: alpine
+    command: ["/bin/sh", "-c", "while /bin/true; do echo hello $(date); done" ]
+  - name: write-and-rotate-logs
+    image: linuxkit/logwrite:b43f735d365d39539013d1678d8cb87a8bceb1a5
+  - name: check-the-logs
+    image: alpine:3.7
+    binds:
+      - /check.sh:/check.sh
+      - /dev/console:/dev/console
+      - /var/log:/var/log
+    command: ["sh", "./check.sh"]
+  - name: poweroff
```

Comment:
Any reason you don't powerdown in `check.sh`? Otherwise the test always runs for 30 seconds even if it succeeds earleir